backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap_classic.cu

```diff
@@ -168,9 +351,19 @@ void scratch_cuda_programmable_bootstrap_64(
     uint32_t input_lwe_ciphertext_count, uint32_t max_shared_memory,
     bool allocate_gpu_memory) {
 
-  if (has_support_to_cuda_programmable_bootstrap_cg<uint64_t>(
-          glwe_dimension, polynomial_size, level_count,
-          input_lwe_ciphertext_count, max_shared_memory))
+#if (CUDA_ARCH >= 900)
+  if (has_support_to_cuda_programmable_bootstrap_tbc<uint64_t>(
+          input_lwe_ciphertext_count, glwe_dimension, polynomial_size,
+          level_count, max_shared_memory))
+    scratch_cuda_programmable_bootstrap_tbc<uint64_t, int64_t>(
+        stream, (pbs_buffer<uint64_t, CLASSICAL> **)buffer, glwe_dimension,
+        polynomial_size, level_count, input_lwe_ciphertext_count,
+        max_shared_memory, allocate_gpu_memory);
+  else
+#endif
+      if (has_support_to_cuda_programmable_bootstrap_cg<uint64_t>(
```

Comment:
Same here

Reply:
The first part of the conditional is within the `#if` conditional. clang is aligning this if with the else at line 327. If I change that `make fmt_gpu` will change back.